tests/vector/utils.py

```diff
@@ -75,10 +75,10 @@ class CustomSpace(gym.Space):
     """Minimal custom observation space."""
 
     def sample(self):
-        return "sample"
```

Comment:
Was the old version of this custom space ever used? Does changing this change any other tests?